adafruit_atecc/adafruit_atecc_asn1.py

```diff
@@ -91,7 +91,15 @@ def get_signature(signature, data):
 
 
 # pylint: disable=too-many-arguments
-def get_issuer_or_subject(data, country, state_prov, locality, org, org_unit, common):
+def get_issuer_or_subject(
```

Comment:
Interesting!  You totally typed this correctly per how it's used in the examples, but digging into this it looks like we do `extend()` with a `bytearray` object to add the string.  This works in Micro/CircuitPython because strings implement the buffer protocol under the hood, but that's not actually the case in CPython.  This should probably be fixed since I don't this this actually works on Blinka right now.

The type annotations is correct though, so let's leave that out of the scope of this PR.

Reply:
I didn't cross reference them at all. Umm what example? lol.  I completely guessed at the types based on the name of the arguments. Buffer types are over my head currently. I don't know the syntax formatting it wants.

Reply:
You can sometimes find uses of the functions either elsewhere in the library or [in one of the example files](https://github.com/adafruit/Adafruit_CircuitPython_ATECC/tree/main/examples).

Buffer types are basically just a string of bytes under the hood.  `str` is this in Micro/CircuitPython because simplicity, but in CPython this isn't the case.  It basically means you *can* do wacky hacky things with strings in CircuitPython that you can't do in CPython, which just means we have to be careful not to do anything that would be impossible on something like a Raspberry Pi.

The syntax formatting it's showing is what `black` changes to for functions with a bunch of stuff in the arguments parts.  It basically splits it over many lines so it's more readable.  They are equivalent:

```python
def func(arg_a: int, arb_b: str, arg_c: float) -> None:
    ...

def func(
    arg_a: int,
    arg_b: str,
    arg_c: float,
) -> None:
    ...
```

The only difference is the comma added after `arg_c` (which could have been in the original even) which is to multiline items like this for readability.